src/app/modules/user/components/project-page/project-page.component.scss

```diff
@@ -0,0 +1,186 @@
+@import 'base/variables';
+@import 'base/fonts';
+
+.projectPage{
+    max-width: 100%;
+    margin-top: 10px;
+    padding: 40px;
+    @media(max-width: 420px) {
+        padding: 10px;
+    }
+
+    background: #FCFCFC;
+    border-radius: 5px 5px 0px 0px;
+
+    h1 {
+        font-family: Gotham Pro;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 35px;
+        line-height: 40px;
```

Comment:
there was a class for this font, include the class name in the HTML class. You can find the class at stylseheets / _fonts.scss